test/phoenix_live_view/integrations/components_test.exs

```diff
@@ -167,6 +167,26 @@ defmodule Phoenix.LiveView.ComponentTest do
                assert_receive {:DOWN, ^ref, _, _, _}
              end) =~ "** (ArgumentError) missing required :id in send_update"
     end
+
+    test "warns if component doesn't exist", %{conn: conn} do
+      {:ok, view, _html} = live(conn, "/components")
+
+      assert ExUnit.CaptureIO.capture_io(:stderr, fn ->
+               send(view.pid, {:send_update, [{StatefulComponent, id: "nemo", name: "NEW-nemo"}]})
+               refute_receive {:updated, _}
```

Comment:
```suggestion
               render(view)
               refute_receive {:updated, _}
```

Otherwise you can have race conditions (i.e. the warning may not have been printed yet). So let's use `render` to "sync" with the live view.

Reply:
Ah! I mostly included the `refute_receive` to sync. This is nicer. But I guess it's also nice to have the refute in there to be clear that in addition to warning, we also don't update.